security-header/README.md

```diff
@@ -4,7 +4,22 @@ Get an **A+** on [security header](https://securityheaders.com/) out of the box.
 
 Before I was using CloudFlare worker (5$ per month) to get the same result. I asked myself, how does the ‘real server’ are configured to manage the scurity-header? This is how.
 
-**screenshot**:
+# Warning
+
+Test those default values locally as they could prevent your website functioning normally.
+
+- Content-Security-Policy	Content Security Policy is an effective measure to protect your site from XSS attacks. By whitelisting sources of approved content, you can prevent the browser from loading malicious assets. Analyse this policy in more detail. You can sign up for a free account on Report URI to collect reports about problems on your site.
+- Feature-Policy	Feature Policy is a new header that allows a site to control which features and APIs can be used in the browser.
+- Referrer-Policy	Referrer Policy is a new header that allows a site to control how much information the browser includes with navigations away from a document and should be set by all sites.
+- Strict-Transport-Security	HTTP Strict Transport Security is an excellent feature to support on your site and strengthens your implementation of TLS by getting the User Agent to enforce the use of HTTPS.
```

Comment:
.... buuuuut they risk bricking their site if they ever don't use HTTPS -- be sure to mention the actual warnings! Don't just describe the feature.